teps/0092-scheduling-timeout.md

```diff
@@ -45,8 +49,22 @@ authors:
 
 ## Summary
 
-Add runtime configuration options to allow setting a timeout for `TaskRuns` which applies before the `TaskRun's`
-underlying pod is scheduled, and only start our existing timeout counters after this timeout has passed, if specified.
+This TEP proposed adding runtime configuration options to allow setting a timeout for `TaskRuns` which would have 
+applied before the `TaskRun's` underlying `Pod` is scheduled, then we would start our existing timeout counters after
+this new timeout had passed, if specified.
+
+However, we decided to pursue supporting queueing directly instead of adding a queueing/scheduling timeout because:
+- the way that queueing timeout works in different scenarios is complicated and the way it interacts with other 
+  timeouts from `PipelineTasks` and `PipelineRuns` (through `TaskRunSpecs` and `TaskRunTemplate`) is even more 
+  complicated - see [attempt][948] to clarify the design
+- the main motivation for this proposal was to address queueing TaskRuns in resource-constrained environments; this 
+  problem will be best addressed through a queueing solution instead of relying on a timeout
```

Comment:
What does the `queueing` solution allude to here? Is it a separate field similar to `scheduling` at the pipelineRun authoring time? Is it a separate controller which watches over the load of a `node` and queues the resources?